components/navigation.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Docs</title>
+    <!-- <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/fontawesome.min.css" integrity="sha512-OdEXQYCOldjqUEsuMKsZRj93Ht23QRlhIb8E/X0sbwZhme8eUw6g8q7AdxGJKakcBbv7+/PX0Gc2btf7Ru8cZA==" crossorigin="anonymous" /> -->
+    
+    <link href="/main.css" rel="stylesheet" />
+    <link href="/css/navigation.css" rel="stylesheet" />
+    
+</head>
+<body>
+    <div id="page">
+      <header>
+        <div class = "header">
+            
+        <h1  class="head-logo-padd head-color" id="logo">
+        AASAN-UI</h1>
+        <h3  class="head-padd head-color">
+        <a class="link" href="/index.html">Home</a></h3>
+        <h3 class="head-padd ">Docs</h3>
+        <h3  class="head-padd head-color">
+        <a class="link" href="/about.html">About</a></h3>
+        </header>
+
+          
+        <nav>
+          <ul>
+            <div class="nav-li">
+            <a class="link" href="/components/alert.html">
+                Alert</a>
+            </div>
+            <div class="nav-li">
+              <a class="link" href="/components/avatar.html">
+                Avatar</a></div>
+            <div class="nav-li">
+              <a class="link" href="/components/badge.html">
+                Badge</a></div>
+            <div class="nav-li">
+              <a class="link" href="/components/button.html">
+                Button</a></div>
+            <div class="nav-li">
+              <a class="link" href="/components/card.html">
+                Card</a></div>
+            <div class="nav-li">
+              <a class="link" href="/components/image.html">
+                Image</a></div>
+            <div class="nav-li">
+              <a class="link" href="/components/input.html">
+                Input</a></div>
+            <div class="nav-li">
+              <a class="link" href="/components/list.html">
+                List</a></div>
+            <div class="nav-li">
+              <a class="link" href="/components/modal.html">
+                Modal</a></div>
```

Comment:
using `<li> `tag instead of `<div>` should be better n.... am I missing something?

Reply:
okay

Reply:
I used div as I was considering them as small containers

Reply:
Then what's the use of 'ul' tag ?

Reply:
Okay, I have removed the 'ul' tag